docs/user/reference/environment-variables.rst

```diff
@@ -56,6 +56,31 @@ All :doc:`build processes </builds>` have the following environment variables au
     :Example: ``https://docs.readthedocs.io/ja/stable/``
     :Example: ``https://example--17.org.readthedocs.build/fr/17/``
 
+.. envvar:: READTHEDOCS_REPOSITORY_URL
+
+    URL from where the source code to generate the documentation is clonned.
+    It could be HTTPS, SSH or any other URL scheme supported by Git.
+    This is the same URL defined under :guilabel:`Admin` > :guilabel:`Settings` > :guilabel:`Repository URL`.
+
+    :Example: ``https://github.com/readthedocs/readthedocs.org``
+    :Example: ``git@github.com:readthedocs/readthedocs.org.git``
+
+.. envvar:: READTHEDOCS_REPOSITORY_IDENTIFIER
+
+    Git identifier checked out from the repository URL.
+    It could be a branch or tag name.
```

Comment:
Updated :arrow_up:

Reply:
I want to update it again :see_no_evil:

Reply:
He he, well... the commit hash cannot _change_ but can be deleted via a force-push. There is no guarantees with Git 😄

Reply:
No, I don't want to update it anymore. I left it _unspecified_ whether the branch/tag name is used for "check out" since it's difficult to write in an elegant way.

Reply:
Yeah, you are write, it became to diffuse, removed all of it now :)